Apps/Inc/Tasks.h

```diff
@@ -31,10 +31,11 @@
 #define TASK_READ_TRITIUM_PRIO              3
 #define TASK_SEND_TRITIUM_PRIO              4
 #define TASK_READ_CAR_CAN_PRIO              5
-#define TASK_UPDATE_DISPLAY_PRIO            6
-#define TASK_SEND_CAR_CAN_PRIO              7
-#define TASK_DEBUG_DUMP_PRIO                8
-#define TASK_COMMAND_LINE_PRIO              9
+#define TASK_SEND_CAR_CAN_PRIO              6
+#define TASK_PUT_IOSTATE_PRIO               7
+#define TASK_UPDATE_DISPLAY_PRIO            8
```

Comment:
I kind of think updating the display is more important than sending controls telemetry, but sendcan is needed for ignition (?).

I think maybe it should be sendcan=6,display=7,iostate=8 but it likely doesn't really matter, this is kinda nitpicky

Reply:
I see what you mean. Even though ignition sequence hinges upon IOState, it's not as important once the car has started up.